protected/humhub/modules/space/models/Space.php

```diff
@@ -116,8 +117,8 @@ public static function tableName()
     public function rules()
     {
         $rules = [
-            [['join_policy', 'visibility', 'status', 'auto_add_new_members', 'default_content_visibility'], 'integer'],
-            [['name'], 'required'],
+            [['join_policy', 'visibility', 'status', 'sort_order', 'auto_add_new_members', 'default_content_visibility'], 'integer'],
+            [['name', 'sort_order'], 'required'],
```

Comment:
Not sure we need this rule here. This may break Space creation, when this field is required but not present.

Reply:
It cannot be no present as I've added a default value in the DB:
![image](https://user-images.githubusercontent.com/23310825/235342729-3206bb29-b54b-4bff-bed0-e41372d3a537.png)
Should I still delete them?